docs(typo): correcting typos in documentation (#112)
dasch-swiss/dsp-tools: docs/dsp-tools-create.md

*gui-elements/gui_attributes*:

> - `Searchbox`: Must be used with _hasLinkTo_ properties. Allows searching and enter a resource that the given resource

Reviewer: ```suggestion
- `Searchbox`: Has to be used with _hasLinkTo_ property. Allows searching resources by entering a resource that the given resource
```

  should link to. It has one gui_attribute that indicates how many properties of the found resources should be

>   indicated. It's mandatory!

Reviewer: ```suggestion
  indicated. This is mandatory.
```
